src/libraries/sendtohelixhelp.proj

```diff
@@ -187,6 +187,21 @@
         <TestTimeout>$(_workItemTimeout)</TestTimeout>
         <LaunchTimeout>$(_workItemTimeout)</LaunchTimeout>
       </XHarnessAppBundleToTest>
+
+      <!-- Create work items for run-only apps -->
+      <XHarnessAppBundleToTest Include="$([System.IO.Directory]::GetDirectories('$(TestArchiveRoot)runonly', '*.app', System.IO.SearchOption.AllDirectories))" >
+        <Targets Condition="'$(TargetArchitecture)' == 'arm'">ios-device</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'arm64'">ios-device</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'x64'">ios-simulator-64</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'x86'">ios-simulator-32</Targets>
+
+        <TestTimeout>$(_workItemTimeout)</TestTimeout>
```

Comment:
If this is something only for the sample apps, would prefer specifying the minimal timeout we know works well. If it fails for any reason, we could essentially want to figure out why it exceeded the timeout, correct ?

Reply:
I can remove it, I guess in this case the helix sdk passes in some default timeout value

Reply:
Removed